src/Scripting/Core/ObjectFormatter.cs

```diff
@@ -437,11 +453,17 @@ public override string ToString()
 
         #region Language Specific Formatting
 
+        /// <summary>
+        /// Returns null if the type is not considered primitive in the target language.
+        /// </summary>
         private string FormatPrimitive(object obj, bool quoteStrings, bool includeCodePoints, bool useHexadecimalNumbers)
```

Comment:
Where is this called?  Can we get rid of this method and remove the C# FormatLiteral(DateTime)?